sdk/appconfiguration/app-configuration/src/index.ts

```diff
@@ -237,7 +237,7 @@ export class AppConfigurationClient {
   setConfigurationSetting(
     key: string,
     configSettings: SetConfigurationSettingConfig,
-    options: SetConfigurationSettingOptions = {}
+    options: SetConfigurationSettingOptions = { label: undefined }
```

Comment:
If a user passed in an empty object for options, would this no longer work?

Reply:
Yes...and now I'm thinking now I should just do this inside of the function instead since this also breaks in another common use case (actually passing in an options struct with some values intact).

Reply:
Fixed by just _always_ setting the options.label field (we had code that did the assignment but only if it was defined in the config setting parameter). 

So now it's way less gross (but I'd still like to fix it earlier in the generated code if possible)